### PR TITLE
ADEN-9677 Bump design-system.

### PR DESCRIPTION
**Pull request by jacekbj**
The bump includes link for US Privacy Do Not Sell link.
This link can be found in page footer.
It is visible only in US California.

### CODE REVIEW

File: includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php [hunk 1]
```diff
@@ -368,6 +368,18 @@ private function getWikiaOrgModel() {
 			];
 		}
 
+		if ( true ) {
```
Comment (rybmat) [inline includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php]: remember to replace it with valid condition ;)

File: includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php [hunk 2]
```diff
@@ -533,6 +545,18 @@ private function getCommunity() {
...
+			$data['links'][] = [
```
Comment (rybmat) [inline includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php]: why are you adding this link twice?

Comment (jacekbj) [reply includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php]: Once for *fandom.com and once for *.wikia.org
It's not decided yet if we want to display it on *.wikia.org. My guess is "No". When the decision is made I will remove/leave it.

Comment (jacekbj) [reply includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php]: Link will appear on both fandom.com and wikia.org.

**Pull request merged**